wallet_transactions/transactions/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+
+from wallets.models import Wallet
+
+# Create your models here.
+
+
+class Transaction(models.Model):
+    """Describe table Transaction"""
+
+    sender = models.ForeignKey(Wallet, on_delete=models.CASCADE, related_name="sender")
+    receiver = models.ForeignKey(
+        Wallet, on_delete=models.CASCADE, related_name="receiver"
+    )
+    transfer_amount = models.DecimalField(max_digits=10, decimal_places=2)
+    commission = models.DecimalField(max_digits=10, decimal_places=2)
+    status = models.CharField(max_length=25)
```

Comment:
we have only 2 possible statuses.
Please, create `STATUS_TYPE` variable, as you did for the wallet model. and add `choices=` attr for the field

Reply:
add STATUS_TYPE and add choices= attr for the field